### PR TITLE
Still more astyling.

### PR DESCRIPTION
**Pull request by kevingranade**
Another batch of astyling low-hanging fruit.

### CODE REVIEW

File: src/projectile.cpp [hunk 1]
```diff
@@ -108,22 +108,22 @@ void apply_ammo_effects( const tripoint &p, const std::set<std::string> &effects
     if( effects.count( "NAPALM" ) > 0 ) {
         g->explosion( p, 4, 0.7, true );
         // More intense fire near the center
-        for( auto && pt : g->m.points_in_radius( p, 1, 0 ) ) {
+        for( auto &&pt : g->m.points_in_radius( p, 1, 0 ) ) {
```
Comment (codemime) [inline src/projectile.cpp]: It's not an issue of astyling or something, but why use rvalue references here? Can't they just be replaced with `const auto &pt`?

File: tests/new_character_test.cpp [hunk 2]
```diff
@@ -119,17 +106,18 @@ TEST_CASE( "starting_items" ) {
...
 
-    auto add_failure = [&]( const profession &prof, const std::vector<trait_id> &traits,
-                            const std::string &item_name, const std::string &reason ) {
-        if( !std::any_of( failures.begin(), failures.end(), [&item_name]( const failure &f ) {
+    auto add_failure = [&]( const profession & prof, const std::vector<trait_id> &traits,
+                            const std::string & item_name, const std::string & reason ) {
+        if( !std::any_of( failures.begin(), failures.end(), [&item_name]( const failure & f ) {
             return f.item_name == item_name;
-            } ) ) {
+        } ) ) {
```
Comment (codemime) [inline tests/new_character_test.cpp]: Multi-line lambdas in conditions still look terrible. Would be better if use a temp variable.

File: tests/vehicle_turrets.cpp [hunk 3]
```diff
@@ -34,16 +36,18 @@ const vpart_info *biggest_tank( const ammotype ammo ) {
         }
     }
 
-    if( res.empty() ) { 
+    if( res.empty() ) {
         return nullptr;
     }
 
-    return * std::max_element( res.begin(), res.end(), []( const vpart_info *lhs, const vpart_info *rhs ) {
+    return * std::max_element( res.begin(), res.end(), []( const vpart_info * lhs,
```
Comment (codemime) [inline tests/vehicle_turrets.cpp]: I'd move both introducer and declarator to the next line:
```c++
return * std::max_element( res.begin(), res.end(), 
[]( const vpart_info * lhs, const vpart_info * rhs ) {
...
```

File: src/projectile.cpp (hunk 1 above)
Comment (kevingranade) [inline src/projectile.cpp]: Agreed, when we started allowing c++11 features, a few people went a bit crazy with some of the new features, foremost amoung them were rvalue refrences/move operations and auto.  At some point I'm going to do a series of code audits and make a lot of this stuff more straightforward.

File: tests/new_character_test.cpp (hunk 2 above)
Comment (kevingranade) [inline tests/new_character_test.cpp]: And... Its a nested lambda, just no.
I'll refactor this.

File: tests/vehicle_turrets.cpp (hunk 3 above)
Comment (kevingranade) [inline tests/vehicle_turrets.cpp]: Good catch, thats a lot better.

File: src/projectile.cpp (hunk 1 above)
Comment (Coolthulhu) [inline src/projectile.cpp]: This one is actually me, I just didn't know what should be used where.

**Pull request merged**